From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Boleslaw Tokarski <boleslaw.tokarski@jollamobile.com>
Date: Thu, 11 Feb 2021 16:01:11 +0200
Subject: [PATCH] Align watch with what is in procps-ng

---
 procps/watch.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/procps/watch.c b/procps/watch.c
index 059eb1dda..3ea55fd81 100644
--- a/procps/watch.c
+++ b/procps/watch.c
@@ -14,7 +14,7 @@
 //config:	watch is used to execute a program periodically, showing
 //config:	output to the screen.
 
-//applet:IF_WATCH(APPLET(watch, BB_DIR_BIN, BB_SUID_DROP))
+//applet:IF_WATCH(APPLET(watch, BB_DIR_USR_BIN, BB_SUID_DROP))
 
 //kbuild:lib-$(CONFIG_WATCH) += watch.o
 
-- 
2.29.2

